button-examples/Android/android-button-example.md

```diff
@@ -169,6 +372,19 @@ toggleButton.addOnButtonCheckedListener { toggleButton, checkedId, isChecked ->
 }
 ```
 
+#### Key properties
```

Comment:
Should there be a rel. link to the outlined button, or should there be an additional table for any `button` settings?